lms/static/sass/bootstrap/_layouts.scss

```diff
@@ -1,8 +1,10 @@
 // LMS layouts
 
 .content-wrapper {
+    margin-top: $baseline;
```

Comment:
@HarryRein We should discuss this. My preference is to make the skeleton such that a user can build a new page without having to mess with margins. Not doing it is more brittle because it means every page comes up with its own mechanism, and if we want to tweak it we'll have to change every page. We don't have many Bootstrap pages yet, so I'd rather get this right out-of-the-gate.